### PR TITLE
Feature/landing template

### PR DESCRIPTION
**Pull request by AshGDS**
Hi @punalsc 

Here is the landing page for the Latin revamp as a WordPress child theme.

Thanks

### CODE REVIEW

File: inc/functions-child.php [hunk 1]
```diff
@@ -15,4 +15,40 @@ function dequeue_parent_style()
...
+				</div>
+				<div class="entry-content">
+					<?php if ( $box['image'] ) { ?>
+						<a href="<?php echo $box['url']; ?>" class="thumbnail" title="<?php echo $box['title'] ?>">
+							<img src="<?php echo $box['image']; ?>" class="img-responsive" alt="<?php echo $box['title'] ?>">
+						</a>
+					<?php } ?>
+					<p><?php echo $box['excerpt']; ?></p>
+					<?php if ( isset($box['child_pages']) ) { ?>
+						<ul >
```
Comment (sdwilkes) [inline inc/functions-child.php]: Ln 39 - add class="full" to the UL to use the bulleted navigation style (used for lists of links).

File: page-home.php [hunk 2]
```diff
@@ -7,25 +7,104 @@
...
+	                           the_content();
+
+                            endwhile;
+                            ?>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+        <div class="row equal-heights" id="equal-heights">
```
Comment (sdwilkes) [inline page-home.php:44]: Equal height doesn't appear to be working. Please check it's correctly configured.

**Review by sdwilkes — CHANGES_REQUESTED** [review]
Looks good. Just a couple of small issues.

File: page-home.php (hunk 2 above)
Comment (domjngo) [reply page-home.php:44]: @AshTNA we need to add similar code to the theme https://github.com/nationalarchives/tna-base/blob/develop/inc/functions/tna-functions.php#L20

File: inc/functions-child.php (hunk 1 above)
Comment (AshGDS) [reply inc/functions-child.php]: Hi Simon, this has been done

File: page-home.php (hunk 2 above)
Comment (AshGDS) [reply page-home.php:44]: Hi Simon, this has been done

File: page-home.php [hunk 3]
```diff
@@ -7,25 +7,104 @@
...
-				endwhile;
+				global $post;
+				$image      = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'full-page-width', true ) );
+				$title      = get_the_title();
+				get_page_banner( 'level one', $title, $image, '', '', '' );
 				?>
-			</main>
+            </div>
+        </div>
+        <div class="row">
```
Comment (domjngo) [inline page-home.php]: We are missing the main element. Wrap the follow two rows with `<main>`

**Review by domjngo — CHANGES_REQUESTED** [review]

File: page-home.php (hunk 3 above)
Comment (AshGDS) [reply page-home.php]: This has been done

**Review by domjngo — APPROVED** [review]

**Pull request merged**